### ISSUE TITLE
Add keras applications backbones

### ISSUE THREAD

**Issue by jalabort** [jalabort/nucleus#6]
Add the following Keras applications backbones:
* `MobileNetV2`
* `NASNetMobile`
* `DenseNet121`
* `DenseNet169`
* `DenseNet201`
* `Xception`
* `InceptionV3`
* `ResNetXt50`

**Comment by jalabort** [comment]
All the above Keras backbones have been added. However, only `MobileNet` and `MobileNetV2` have been properly tested.

Doing this work it was noted that Keras implements `batch normalization` layers in a pretty [controversial manner](https://github.com/keras-team/keras/pull/9965). As a result this [patch](https://github.com/keras-team/keras/pull/9965#issuecomment-501933060) is applied to all `backbones`. For a very detailed explanation of the issue, please refer to this excellent [blog post](http://blog.datumbox.com/the-batch-normalization-layer-of-keras-is-broken).
